index.html

```diff
@@ -107,6 +108,53 @@
   - The server and/or any relevant intermediaries are in full control of which metrics are communicated to the user agent and when. For example, access to some metrics may be restricted due to privacy or security reasons - see <a href="#privacy-and-security"></a> section.
 </div>
 
+To <dfn>parse a server-timing header field</dfn> given a string |field|:
+
+1. Let |position| be a [=string/position variable=], initially pointing at the start of |field|.
+
+1. Let |rawName| be the result of [=collecting a sequence of code points=] that are not equal to U+003B (;), given |position|.
+
+1. Let |name| be the result of [=strip leading and trailing ASCII whitespace|stripping=] |rawName|.
```

Comment:
The algorithm is defined as operating in place rather than returning a result.

Reply:
Fixed